src/TaskManager/Plug-ins/Argo/Logging/Log.cs

```diff
@@ -51,5 +51,8 @@ public static partial class Log
 
         [LoggerMessage(EventId = 1014, Level = LogLevel.Information, Message = "{artifactName} in template {templateName} is not configured as no matching input information was found.")]
         public static partial void NoInputArtifactStorageConfigured(this ILogger logger, string artifactName, string templateName);
+
+        [LoggerMessage(EventId = 1015, Level = LogLevel.Debug, Message = "metadata file in {path} in bucket {bucket} was not found.")]
```

Comment:
[LoggerMessage(EventId = 1015, Level = LogLevel.Debug, Message = "metadata file in {path} in bucket {bucket} was not found.")]
 capital missing start of message

Reply:
Fixed